examples/tutorials/assets/dft_tutorial.yaml

```diff
@@ -0,0 +1,23 @@
+- e_type : 'ae'
```

Comment:
We need to add this tutorial to https://github.com/deepchem/deepchem/blob/master/examples/tutorials/website-render-order/5-quantum-chemistry.csv for it to get rendered

Reply:
We also need to add the google colab link.

Reply:
Done.